Startup.cs

```diff
@@ -16,33 +15,16 @@
 using OrchardCore.ContentManagement;
 using OrchardCore.ContentManagement.Display.ContentDisplay;
 using OrchardCore.Data.Migration;
-using OrchardCore.DisplayManagement.Handlers;
 using OrchardCore.Modules;
-using OrchardCore.Navigation;
 using OrchardCore.ResourceManagement;
-using OrchardCore.Security.Permissions;
-using OrchardCore.Settings;
 using OrchardCore.Users.Events;
-using OrchardCore.Users.Models;
-using OrchardCore.Users.Services;
 using OrchardCore.Workflows.Helpers;
 using System;
+using System.Threading.Tasks;
 
 namespace Lombiq.Privacy
 {
-    [Feature(FeatureNames.Module)]
     public class Startup : StartupBase
-    {
-        public override void ConfigureServices(IServiceCollection services)
-        {
-            services.AddScoped<IPermissionProvider, PrivacyConsentPermissions>();
-            services.AddScoped<INavigationProvider, AdminMenu>();
```

Comment:
Override-olni lehet a menüt, szóval lehet icon (ez csak példa kód):
![image](https://user-images.githubusercontent.com/1976647/135001036-f4555140-e51f-4b02-b0c0-839efbcc1207.png)

Reply:
Ez eredetileg úgy próbáltam  megcsinálni hogy csináltam mindegyik featurenek egy egy `INavigationProvider` implementációt és a saját Startup-jában adtam hozzá `.AddScoped()`-dal és ebben az esetben nem müködött. Most újra próbáltam és most sem müködik ha a "saját" startup-jába teszem. Ez most ahogy nézem azért van mert a `Mainfest`-ben az `OrchardCore.CustomSettings`-et dependecievé tettem a featureokhoz. Ez kicsit logikátlan hogy így nem müködik de én azt tapasztaltam hogy ha ezt kiszedem akkor akkor overrideolható lesz. DE ezt hogy kivettem dependencie-ből csak azért tehettem meg mert már egy olyan DB-vel próbáltam ahol a `CustomSettings`-et már bekapcsoltam.

Tovább vizsgálódva Krisztiánnal arra jutottunk hogy ez az override csak akkor müködik hogyha a feature Startup-jában override-olom az Ordert(pl -1000-rel azaz előbb futtatom le mint a dependencijét?!!??!!?!). Mindenesetre ha ezt csinálom akkor müködik a custom settings funkcionalitás és overrideolódik az ikon

Akkor oldajam meg ezt mind a 3 feature esetén?

Reply:
Hmm, hát ez fura, mert én semmi ilyen machinációt nem csináltam, hozzáadtam a kódodhoz és kész. Lehet, hogy nem az egyik sub-feature Startupjában regisztráltam, hanem a rootban, de max ennyi lehet (a kód már nincs meg).

De Orderrel is jó lesz, ja.